spark-commons/src/main/scala-spark2.4-jvm/za/co/absa/spark/commons/adapters/CallUdfAdapter.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2021 ABSA Group Limited
```

Comment:
2023

Reply:
on many places below as well

Reply:
It should be the repo/project inception year. Not the file creation one.
(Surprised it's 2021 for this one and not 2022 🤔 )

Reply:
oh, okay then, I didn't know